lib/src/module.dart

```diff
@@ -122,6 +123,18 @@ abstract class Module {
           '  Call build() before accessing this.');
   String _uniqueInstanceName;
 
+  static String? _nameValidation(
```

Comment:
Add a doc string describing what this function is for, even for private methods, it's helpful

Reply:
Any template or script is available for doc string documentation? Maybe can create a template on documentation style to make sure it is similar across all the doc.